src/main/java/frc/robot/Robot.java

```diff
@@ -28,15 +32,20 @@
   public static double[] changeAngle = new double[]{};
   public static double[] changeRotations = new double[]{};
   public static boolean Shift;
+  public static DoubleSolenoid lShifter;
+  public static DoubleSolenoid rShifter;
```

Comment:
Nit: This can be defined on the same line with a comma

`public static DoubleSolenoid lShifter, rShifter;`